ansible/roles/grafana/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: mkdir for config and data
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: "u=rwx,g=rx,o="
+  with_items:
+    - "{{ grafana_config_path }}"
+    - "{{ grafana_lib_path }}"
```

Comment:
I am going for the moment defer this to future work and not consider it part of MVP.